scripts/eval_stop_guard.py

```
@@ -0,0 +1,248 @@
+                    status = (event.get("status")
+                              or (event.get("patch") or {}).get("status"))
+                    if status in TERMINAL_STATUSES:
+                        live.pop(tid, None)
+    except OSError:
+        return None
+
+    return live
+
+
+def _state_path(session_id):
+    tmp = os.environ.get("TMPDIR") or "/tmp"
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)